packages/browser/src/index.ts

```diff
@@ -68,6 +70,35 @@ export default class LogtoClient {
     window.location.assign(signInUri);
   }
 
+  public async signOut(postLogoutRedirectUri?: string) {
+    if (!this.idToken || !this.oidcConfig) {
+      return;
+    }
+
+    const { clientId, requester } = this.logtoConfig;
+    const { endSessionEndpoint, revocationEndpoint } = this.oidcConfig;
+    const logtoKey = getLogtoKey(clientId);
+
+    if (this.refreshToken) {
+      void revoke(revocationEndpoint, clientId, this.refreshToken, requester);
```

Comment:
this is not good since the request will be interrupted when `window.location` changed